scripts/local_testnet/README.md

```diff
@@ -94,10 +141,16 @@ beacon node:
 ```
 e.g. to attach to the above created beacon node
 ```bash
-./validator_client.sh $HOME/.lighthouse/local-testnet/node_1 http://localhost:8000
+./validator_client.sh $HOME/.lighthouse/local-testnet/node_1 http://localhost:8001
 ```
 
-You can create additional beacon node and validator client instances with appropriate parameters.
+You can create additional geth, beacon node and validator client instances by changing the ports, e.g., for a second geth, beacon node and validator client:
+
+```bash
+./geth.sh $HOME/.lighthouse/local-testnet/geth_2 7002 6002 5002 genesis.json
+./beacon_node.sh $HOME/.lighthouse/local-testnet/node_2 9002 9102 8002 http://localhost:5002 ~/.lighthouse/local-testnet/geth_2/geth/jwtsecret
+./validator_client.sh $HOME/.lighthouse/local-testnet/node_2 http://localhost:8002
```

Comment:
if you want to run validators on a new VC, it would require generating additional keys for that, i.e. updating `vars.env` and start over.

Reply:
as discussed, by default we generate keys for 4 VCs, so this would be fine